### PR TITLE
Add default wallpaper

### PR DESCRIPTION
**Pull request by elParaguayo**
Trawling through old issues and I noticed a comment about needing a default wallpaper to replace the black screen that accompanies the default config. 

Here's my submission.

Don't hold back on critiquing the artwork!

### CODE REVIEW

File: libqtile/resources/default_config.py [hunk 1]
```diff
@@ -122,6 +123,9 @@
 )
 extension_defaults = widget_defaults.copy()
 
+wallpaper_path = Path(__file__).resolve() / ".." / "qtile-wallpaper.png"
```
Comment (m-col) [inline libqtile/resources/default_config.py:126]: Maybe stick it in resources/ alongside the default config?

**Comment by m-col** [comment]
I love the wallpaper, so +1 from me.

File: libqtile/resources/default_config.py (hunk 1 above)
Comment (elParaguayo) [reply libqtile/resources/default_config.py:126]: That's exactly where it is.

Comment (m-col) [reply libqtile/resources/default_config.py:126]: Oh yeah, brainfart :D

**Comment by ramnes** [comment]
Won't this wallpaper confuse users as to what is a real window or not? :D

**Comment by m-col** [comment]
> Won't this wallpaper confuse users as to what is a real window or not? :D

Lol really lacking faith in humanity huh? I think it's fine: 
image

Of course I think everyone has to be completely happy with it if we're going to have a default wallpaper so no worries if you'd prefer not to :)

**Pull request closed without merging**